README.md

```diff
@@ -32,6 +32,27 @@ python -m pip install -r <(curl -s https://raw.githubusercontent.com/sigstore/si
 
 This installs the requirements file located [here](https://github.com/sigstore/sigstore-python/blob/main/install/requirements.txt), which is kept up-to-date.
 
+### GitHub Actions
+
+`sigstore-python` has [an official GitHub Action](https://github.com/trailofbits/gh-action-sigstore-python)!
```

Comment:
Not sure I can call it official since it hasn't been transferred to sigstore yet.

Reply:
IMO it's fine, if only to avoid churn when we eventually migrate it.